js/interface.js

```diff
@@ -37,6 +37,7 @@ var isShowingAll = false;
 var columns;
 var dataSourcesToSearch = [];
 var initialLoad = true;
+var includeOrExclude = 'include';
```

Comment:
This variable can be named `columnsListMode`

Reply:
@squallstar changes done as per suggestion